programs/drift/src/instructions/keeper.rs

```diff
@@ -959,6 +1103,26 @@ pub fn handle_settle_pnl<'c: 'info, 'info>(
         .map(|_| ErrorCode::InvalidOracleForSettlePnl)?;
     }
 
+    if state.builder_codes_enabled() || state.builder_referral_enabled() {
+        if let Some(ref mut escrow) = builder_escrow {
+            escrow.revoke_completed_orders(user)?;
```

Comment:
does it make sense to revoke after the sweep?

Reply:
i dont think so since sweep only settles RevenueShareOrders when they're completed, and they're only considered completed after `revoke_completed_orders` is run.